.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__
+ext/dcos-installer/packages/
```

Comment:
Gen makes that packages/ directory in the dcos-installer dir when I execute it locally in my dev env. @spahl has seen me accidentally push .tar.xz files that were placed in it in the past, and I almost did it again testing this PR. Would like to leave it here so we don't repeat it.